backend-docker/pier/pier.go

```diff
@@ -163,4 +164,22 @@ func (p *Pier) GetJob(jobID JobID) (Job, error) {
 		return job, def.Err(nil, "not found")
 	}
 	return job, nil
+}
+
+// CopyFromVolume exported
+func (p *Pier) CopyFromVolume(volumeID string, filePath string) (dckr.ContainerID, error){
```

Comment:
The pier library should not return any container ids. Maybe this function should be merged with DownstreamContainerFile?